frontend/src/components/my-grant-profile-page-handler/no-grant.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Button,
+  Container,
+  Heading,
+  Spacer,
+  Text,
+} from "@python-italia/pycon-styleguide";
+import { FormattedMessage } from "react-intl";
+
+import { useCurrentLanguage } from "~/locale/context";
+import { DeadlineStatus, type MyProfileWithSubmissionsQuery } from "~/types";
+
+import { createHref } from "../link";
+
+type Props = {
+  deadline: MyProfileWithSubmissionsQuery["conference"]["deadline"];
+};
+
+export const NoGrant = ({ deadline }: Props) => {
+  const deadlineStatus = deadline.status;
+  const language = useCurrentLanguage();
+
+  return (
+    <Container size="medium" center={false} noPadding>
+      <Heading size={2}>
+        <FormattedMessage id="profile.myGrant.noGrant.heading" />
+      </Heading>
+      <Spacer size="small" />
+      <Text size={2}>
+        {deadlineStatus === DeadlineStatus.HappeningNow && (
+          <FormattedMessage id="profile.myGrant.noGrant.body.canSubmit" />
+        )}
+        {deadlineStatus === DeadlineStatus.InThePast && (
+          <FormattedMessage id="profile.myGrant.noGrant.body.closed" />
+        )}
+        {deadlineStatus === DeadlineStatus.InTheFuture && (
+          <FormattedMessage id="profile.myGrant.noGrant.body.openingSoon" />
+        )}
+      </Text>
+      <Spacer size="large" />
```

Comment:
this spacer will be rendered regardless of the deadline status meaning that if deadlineStatus is `InThePast` we will have an extra unused space